### PR TITLE
126 geospatialworldpop count

### PR DESCRIPTION
**Pull request by rowannicholls**
- Edited `process_gadm_worldpopcount()`
- Edited `test_plot_gadm_macro_heatmap()`

To be reviewed and merged after #124

### CODE REVIEW

**Comment by abhidg** [comment]
@rowannicholls can you rebase onto main and squash some of the commits? thanks!

**Comment by rowannicholls** [comment]
> @rowannicholls can you rebase onto main and squash some of the commits? thanks!

This ticket followed #124 so now that that has been merged in I have pulled in those changes

File: src/dart_pipeline/process.py [hunk 1]
```diff
@@ -155,6 +148,151 @@ def process_gadm_admin_map_data(iso3: str, admin_level: AdminLevel):
...
+                new_path = source_path(source, Path(iso3, filename))
+                logging.info('importing:%s', new_path)
+                print(f'File {path} not found.')
+                print(f'Importing {new_path} instead')
+                src = rasterio.open(new_path)
+                file_found = True
+                break
+        if not file_found:
+            msg = f'{path}: No such file or directory. Either it has not ' + \
+                'been downloaded or data does not exist for this year.'
```
Comment (abhidg) [inline src/dart_pipeline/process.py:195]: this year or any year since 1990

**Review by abhidg — APPROVED** [review]
Looks good!

**Pull request merged**